README.md

```diff
@@ -1,24 +1,187 @@
-# README
+<a name="readme-top"></a>
```

Comment:
- [ ] Kindly, add a "tests" section on your readme detailing the correct way to run your tests.

In this case, the correct way would be `rake rswag`:

![image](https://github.com/jonas-45/motorcyle-booking/assets/57421823/4e0f21fa-769b-4d35-a00c-cb1dd16cd25d)

This command is idempotent, which means no matter how many times is run, the result will always be the same.  👍

Reply:
Hi @Meltrust ,
Thanks for your time and for reviewing our code. We will update the readme file as you advised.